cmake/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+# TODO(dkorolev): Consider intelligently re-running `cmake` from the `Makefile` if the set of source files has changed.
```

Comment:
NOTE(dkorolev), @mzhurovich, this was tested a long time ago in https://github.com/dkorolev/example_use_current_with_cmake, just cleaned up a bit and moved this into `current`.'